<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartedMenu</class>
 <widget class="QWidget" name="StartedMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Стартовое Меню</string>
  </property>
  <widget class="QLabel" name="Main_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Farmer's Valley</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Play_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Играть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать Сохранение</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
